nais-dev.yaml

```diff
@@ -66,7 +64,10 @@ spec:
     outbound:
       rules:
         - application: esyfovarsel
+          namespace: team-esyfo
```

Comment:
Tror ikke man trenger å spesifisere namespace i AP dersom appene opererer i samme namespace:
https://docs.nais.io/nais-application/access-policy/#send-requests-to-other-app-in-the-same-namespace

Reply:
ah, sant det. fikser.